cpp/src/io/comp/gpuinflate.cu

```diff
@@ -980,27 +980,27 @@ __device__ int parse_gzip_header(uint8_t const* src, size_t src_size)
     {
       uint8_t flags = src[3];
       hdr_len       = 10;
-      if (flags & GZIPHeaderFlag::fextra)  // Extra fields present
+      if (flags & detail::GZIPHeaderFlag::fextra)  // Extra fields present
```

Comment:
looks like another file missing `namespace detail` ? Not for this PR :D

Reply:
Ah yep, it looks like all the header files in `src/io/comp` need to be moved to detail namespace -
```
brotli_dict.hpp
unbz2.hpp
nvcomp_adapter.cuh (?)
nvcomp_adapter.hpp (?)
gpuinflate.hpp
brotli_tables.hpp
```
Haha yes, will defer this to a follow-on PR :D